servicex/templates/did-finder-rucio/deployment.yaml

```diff
@@ -23,38 +23,46 @@ spec:
         command: ["/usr/src/app/runme.sh"]
         imagePullPolicy: {{ .Values.didFinder.rucio.pullPolicy }}
         env:
-    {{- if .Values.secrets }}
+          {{- if .Values.secrets }}
           - name: RMQ_PASS
             valueFrom: 
               secretKeyRef:
                 name: {{ .Values.secrets }}
                 key: rabbitmq-password
           - name: RMQ_URI
             value: amqp://user:$(RMQ_PASS)@{{ .Release.Name }}-rabbitmq:5672/?heartbeat=9000
-    {{- else }}
+          {{- else }}
           - name: RMQ_URI
             value: amqp://user:{{ .Values.rabbitmq.auth.password }}@{{ .Release.Name }}-rabbitmq:5672/?heartbeat=9000
-    {{- end }}
-    {{- if .Values.didFinder.rucio.cachePrefix }}
+          {{- end }}
+          {{- if .Values.didFinder.rucio.cachePrefix }}
           - name: CACHE_PREFIX
             value: "{{ .Values.didFinder.rucio.cachePrefix }}"
-    {{- end }}
-    {{- if .Values.didFinder.rucio.reportLogicalFiles }}
+          {{- end }}
+          {{- if .Values.didFinder.rucio.reportLogicalFiles }}
           - name: REPORT_LOGICAL_FILES
             value: "--report-logical-files"
-    {{- end }}
+          {{- end }}
           - name: RUCIO_LATITUDE
             value: "{{ .Values.didFinder.rucio.servicex_latitude }}"
           - name: RUCIO_LONGITUDE
             value: "{{ .Values.didFinder.rucio.servicex_longitude }}"
           - name: INSTANCE_NAME
             value: {{ .Release.Name }}
+          {{- if ((.Values.didFinder.rucio.memcache).enabled) }}
+          - name: MEMCACHE
+            value: "True" 
+          {{- end }}
         volumeMounts:
           - name: rucio-cfg
             mountPath: /opt/rucio/etc/
           - name: x509-secret
             mountPath: /etc/grid-security-ro
             readOnly: true
+      {{- if ((.Values.didFinder.rucio.memcache).enabled) }}
+      - name: {{ .Release.Name }}-did-finder-rucio-memcache
+        image: memcached:{{ .Values.didFinder.rucio.memcache.tag }}
```

Comment:
Burt has requested that we don't have hardcoded docker images in our charts - he will want to use a local copy of this image so override it in `values.yaml`

Reply:
OK I understand.